modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml

```diff
@@ -12,47 +12,87 @@ servers:
 security:
   - esenders-token: []
 paths:
-  /api/notice/notices/{businessId}/validation-report:
+  /api/notice/notices/{noticeId}/validation-report:
```

Comment:
The current specification uses "businessId", not "noticeId".

Reply:
I think that renaming this parameter @meletev  does not have any impact as it os only a placeholder in the URL. I think we can keep the change proposed by @pcolast